src/metabase/lib/metric/basics.cljc

```diff
@@ -0,0 +1,27 @@
+(ns metabase.lib.metric.basics
```

Comment:
I'm not a big fan of separate like this, I never know where to put things.

Reply:
Believe me I hate it too, but this is to break a dep cycle and I don't really know a better way.